pennylane_lightning/lightning_kokkos/_measurements.py

```diff
@@ -0,0 +1,465 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Class implementation for state vector measurements.
+"""
+
+# pylint: disable=import-error, no-name-in-module, ungrouped-imports
+try:
+    from pennylane_lightning.lightning_kokkos_ops import MeasurementsC64, MeasurementsC128
+except ImportError:
+    pass
```

Comment:
I have already checked it and looks like Codecov does not have a good integration because in [GitHub actions](https://github.com/PennyLaneAI/pennylane-lightning/actions/runs/10151245099/job/28070111328?pr=817#step:12:35) shows that the line `pass` in covered.